src/model.cpp

```diff
@@ -754,7 +754,7 @@ void Model::getObservableSensitivity(gsl::span<realtype> sy, const realtype t,
     fdydx(t, x);
     fdydp(t, x);
 
-    this->sx.assign(nx_solver * nplist(), 0.0);
+    this->sx.resize(nx_solver * nplist());
```

Comment:
This does not reset to 0.0 anymore (unless was empty before). Intended?

Reply:
Yes. The flatten in the next step makes should make setting to 0.0 redundant and resize is hopefully a bit more efficient.